test/unit/event/src/EventTest.cpp

```diff
@@ -195,63 +173,64 @@ BOOST_AUTO_TEST_CASE_TEMPLATE(
+            REQUIRE(!alpaka::event::test(k1));
+            REQUIRE(!alpaka::event::test(e1));
 
             // q1 = [k1, e1, k2]
             alpaka::queue::enqueue(q1, k2);
-            BOOST_REQUIRE_EQUAL(false, alpaka::event::test(k1));
-            BOOST_REQUIRE_EQUAL(false, alpaka::event::test(e1));
-            BOOST_REQUIRE_EQUAL(false, alpaka::event::test(k2));
+            REQUIRE(!alpaka::event::test(k1));
+            REQUIRE(!alpaka::event::test(e1));
+            REQUIRE(!alpaka::event::test(k2));
 
             // wait for e1
             // q2 = [->e1]
             alpaka::wait::wait(q2, e1);
 
             // q2 = [->e1, e2]
             alpaka::queue::enqueue(q2, e2);
-            BOOST_REQUIRE_EQUAL(false, alpaka::event::test(e2));
+            REQUIRE(!alpaka::event::test(e2));
 
             // re-enqueue should be possible
             // q1 = [k1, e1-old, k2, e1]
             alpaka::queue::enqueue(q1, e1);
-            BOOST_REQUIRE_EQUAL(false, alpaka::event::test(k1));
-            BOOST_REQUIRE_EQUAL(false, alpaka::event::test(k2));
-            BOOST_REQUIRE_EQUAL(false, alpaka::event::test(e1));
-            BOOST_REQUIRE_EQUAL(false, alpaka::event::test(e2));
+            REQUIRE(!alpaka::event::test(k1));
+            REQUIRE(!alpaka::event::test(k2));
+            REQUIRE(!alpaka::event::test(e1));
+            REQUIRE(!alpaka::event::test(e2));
 
             // q1 = [k2, e1]
             k1.trigger();
-            BOOST_REQUIRE_EQUAL(true, alpaka::event::test(k1));
-            BOOST_REQUIRE_EQUAL(false, alpaka::event::test(k2));
-            BOOST_REQUIRE_EQUAL(false, alpaka::event::test(e1));
-            BOOST_REQUIRE_EQUAL(false, alpaka::event::test(e2));
+            REQUIRE(alpaka::event::test(k1));
+            REQUIRE(!alpaka::event::test(k2));
+            REQUIRE(!alpaka::event::test(e1));
+            REQUIRE(!alpaka::event::test(e2));
 
             // q1 = [e1]
             k2.trigger();
-            BOOST_REQUIRE_EQUAL(true, alpaka::event::test(k2));
+            REQUIRE(alpaka::event::test(k2));
             alpaka::wait::wait(e1);
-            BOOST_REQUIRE_EQUAL(true, alpaka::event::test(e1));
+            REQUIRE(alpaka::event::test(e1));
             alpaka::wait::wait(e2);
-            BOOST_REQUIRE_EQUAL(true, alpaka::event::test(e2));
+            REQUIRE(alpaka::event::test(e2));
         }
         else
         {
-            BOOST_TEST_MESSAGE("Can not execute test because CU_DEVICE_ATTRIBUTE_CAN_USE_STREAM_MEM_OPS is not supported!");
+            std::cerr << "Can not execute test because CU_DEVICE_ATTRIBUTE_CAN_USE_STREAM_MEM_OPS is not supported!" << std::endl;
         }
     }
 }
+};
 
 //-----------------------------------------------------------------------------
 // github issue #388
-BOOST_AUTO_TEST_CASE_TEMPLATE(
-    waitForEventThatAlreadyFinishedShouldBeSkipped,
-    TDevQueue,
-    TestQueues)
+struct TestTemplateFinishedShouldBeSkipped
```

Comment:
Could you please fix the names of those structs because the first part you skipped is really essential here?

Reply:
Didn't I? I tried to give them consequent suffixes